modules/calib/test/test_cameracalibration.cpp

```diff
@@ -1769,6 +1769,474 @@ void CV_StereoCalibrationTest_CPP::correct( const Mat& F,
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
+///////////////////////////////// Register Cameras ////////////////////////////////////////////////
+
+class CV_CameraRegistrationTest : public cvtest::BaseTest
```

Comment:
Most of the test are written in old style; perhaps, need to rewrite them in GTest-style sometimes, but it's not urgent, of course